conans/client/remote_manager.py

```diff
@@ -240,6 +244,15 @@ def _call_remote(self, remote, method, *argc, **argv):
             logger.error(traceback.format_exc())
             raise ConanException(exc, remote=remote)
 
+    @staticmethod
+    def package_snapshot_complete(snapshot):
+        integrity = True
+        for keyword in ["conaninfo", "conanmanifest", "conan_package"]:
```

Comment:
Why having here the names instead of using `CONANINFO`, `CONAN_MANIFEST`,... variables?

Reply:
Was done this way to allow changing the file extensions in the future. For example, *conan_package.txz* instead of *conan_package.tgz*